src/layman/common/micka/util.py

```diff
@@ -571,3 +572,15 @@ def get_record_element_by_id(csw, id):
     # current_app.logger.info(f"Number of md records id={id}: {len(els)}")
     result = els[0] if len(els) > 0 else None
     return result
+
+
+def get_muuid_from_operates_on_link(operates_on_link):
```

Comment:
Trvalo mi 5 minut pochopit význam "operates_on". Rád bych se spolu pobavil jestli nenajdeme popisnější název.

Reply:
Ch8pu, je to název elementu z metadatového záznamu (opearatesOn).